package.json

```diff
@@ -89,6 +89,8 @@
     "emoji-js": "^3.4.0",
     "eslint-plugin-ember": "^5.2.0",
     "fuzzysort": "^1.0.1",
+    "liquid-fire": "^0.29.5",
+    "liquid-wormhole": "^2.1.5",
```

Comment:
Neat, I hadn't heard of these 👍

Reply:
Yeah, Liquid Fire was a default tool to make Ember apps animated for years. It seems like during the last year it tends to be replaced by [Ember Animated](https://ember-animation.github.io/ember-animated/), which is basically its successor, created by the same author. Although, our current `modal-dialog` component out of the box supports `liquid-fire`, so I decided to go with it :)